### PR TITLE
Fix grammar and markdown format

### PR DESCRIPTION
**Pull request by aasanchez**
## Describe your changes

Just a suggestion to improve the readme readability, and make it a Markdown valid document, also leveraging the Github Markdown Flavoured Markdown

### CODE REVIEW

**Review by krealyt — APPROVED** [review]
Very appreciated, thanks!

Needs final approval by repo maintainer. From my side: Everything fine.

**Review by hikinggrass — APPROVED** [review]

**Pull request merged**
